src/APL/fileAPL.ts

```diff
@@ -0,0 +1,87 @@
+import debugPkg from "debug";
+import { promises as fsPromises } from "fs";
+
+import { APL, AuthData } from "./apl";
+
+const debug = debugPkg.debug("FileAPL");
```

Comment:
@zaiste had a good idea to nest namespaces - what do you think about

Level 1 - namespace `@saleor/app-sdk` or `app-sdk`
Level 2 - namespace for module `FileAPL`

So final log will be `app-sdk:FileAPL: Something Happened`

I link this idea because users can filter either entire package or selected modules